views/default/tab_client_snapshots.pdt

```diff
@@ -13,27 +13,29 @@
                 <tr>
                     <th><?php $this->_('Vultr.tab_client_snapshots.snapshot_id'); ?></th>
                     <th><?php $this->_('Vultr.tab_client_snapshots.description'); ?></th>
+                    <th><?php $this->_('Vultr.tab_client_snapshots.status'); ?></th>
                     <th><?php $this->_('Vultr.tab_client_snapshots.options'); ?></th>
                 </tr>
             </thead>
             <tbody>
                 <?php
-                foreach ($snapshots as $snapshotid => $description) {
+                foreach ($snapshots as $snapshot) {
                 ?>
                 <tr>
-                    <td><strong><?php $this->Html->_($snapshotid); ?></strong></td>
-                    <td><?php $this->Html->_($description); ?></td>
+                    <td><strong><?php $this->Html->_($snapshot->SNAPSHOTID); ?></strong></td>
```

Comment:
Is the ID really in all caps while other object properties aren't?

Reply:
Yes